src/pages/Home.test.js

```diff
@@ -0,0 +1,26 @@
+// Imports React into our test file.
+import React from 'react'
+
+// Imports Enzyme testing and deconstructs Shallow into our test file.
+import Enzyme, { shallow } from 'enzyme'
+
+// Imports Adapter utilizing the latest react version into our test file so we can run a testing render on any component we may need.
+import Adapter from 'enzyme-adapter-react-16'
+
+// Imports in the component we are going to be testing.
+import Home from './Home.js'
+
+//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
+Enzyme.configure({adapter: new Adapter() })
+
+describe("When Home renders", ()=>{
+    let homerRender
+    beforeEach(() => {
+        homeRender = shallow(<Home />)
+    })
+    it("displays a header", ()=>{
+        const homeImageRender = homeRender.find("img")
```

Comment:
I didn't notice an img tag on your Home.js. Please update your Home.js to reflect this img or update your test to reflect what is on your Home.js